app/src/main/java/org/imozerov/streetartview/ui/explore/map/ArtMapFragment.kt

```diff
@@ -32,10 +33,23 @@ class ArtMapFragment : Fragment(), Filterable, ArtView {
                 .commit()
 
         mapFragment.getMapAsync {
+            val infoView : InfoView
+            try{
+                infoView = (activity as InfoView)
+            } catch (cce: ClassCastException) {
+                throw RuntimeException("ExploreArtActivity must implement InfoView interface")
```

Comment:
Do not hardcode activity name here.
Use context.toString() here (it will be implicitly called) if you'll use $context